src/libs/error.ts

```diff
@@ -1,12 +1,14 @@
 import { AnyPromiseFn, GenericPromiseFn, safeCurry } from '@terra-ui-packages/core-utils';
 import _ from 'lodash/fp';
+import { sessionTimedOutErrorMessage } from 'src/auth/auth-errors';
 import { notify } from 'src/libs/notifications';
 
 export const reportError = async (title, obj) => {
   console.error(title, obj); // helpful when the notify component fails to render
   // Do not show an error notification when a session times out.
   // Notification for this case is handled elsewhere.
-  if (obj instanceof Error && obj.message === 'Session timed out') {
+  // For some reason making using sessionTimedOutErrorMessage instead of the raw string breaks tests
```

Comment:
```suggestion
  // For some reason using sessionTimedOutErrorMessage instead of the raw string breaks tests
```
This was a cyclic dependency issue, right?  Maybe we can put that here so that at some point in time in the future, if someone wants to test if that dependency still exists, they can try fixing it.  Or leave a clue so that someone can just try to fix it outright.

Reply:
This is resolved now, so I will remove the comment